terraform/core/47-mwaa.tf

```diff
@@ -0,0 +1,187 @@
+resource "aws_iam_role" "mwaa_role" {
+  name = "mwaa_execution_role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Action = "sts:AssumeRole",
+        Effect = "Allow",
+        Principal = {
+          Service = [
+            "airflow-env.amazonaws.com",
+            "airflow-env.amazonaws.com"
```

Comment:
Not sure this needs to be duplicated.

Reply:
Aha, my fault. I did some formatting and renaming, not sure how I changed this.